mp/src/game/client/momentum/ui/HUD/hud_mapfinished.cpp

```diff
@@ -446,6 +486,50 @@ void CHudMapFinishedDialog::SetCurrentPage(int pageNum)
     m_pZoneVelMax->SetText(CConstructLocalizedString(m_pwVelMax, m_pRunStats ? m_pRunStats->GetZoneVelocityMax(m_iCurrentPage, velType) : 0.0f));
 }
 
+void CHudMapFinishedDialog::LoadPlayerBestTime()
+{
+    const auto pPlayer = C_MomentumPlayer::GetLocalMomPlayer();
+    if (!pPlayer)
+    {
+        return;
+    }
+    const auto pRunData = pPlayer->GetCurrentUIEntData();
+
+    CMomReplayBase *pBase = MomUtil::GetBestTime(g_pGameRules->MapName(), pRunData->m_flTickRate,
+                                                 pRunData->m_iCurrentTrack, pRunData->m_iRunFlags);
+
+    if (!pBase)
+    {
+        return;
+    }
+    
+    m_pPbRunStats->FullyCopyFrom(*pBase->GetRunStats());
+    m_flPbOverallTime = pBase->GetRunTime();
+
+    delete pBase;
+}
+
+void CHudMapFinishedDialog::GetDiffString(float diff, char* compareStringOut, Color* compareColorOut) 
+{
+    char tempANSITimeOutput[BUFSIZETIME];
+    char diffSign;
+
+    diffSign = diff > 0.0f ? '+' : '-';
```

Comment:
Join declaration and assignment.
```suggestion
    char diffSign = diff > 0.0f ? '+' : '-';
```